sylvia/src/multitest.rs

```diff
@@ -39,22 +103,24 @@ impl App {
 }
 
 #[must_use]
-pub struct ExecProxy<'a, 'app, Error, Msg>
+pub struct ExecProxy<'a, 'app, Error, Msg, ExecC = Empty>
 where
-    Msg: Serialize + std::fmt::Debug,
-    Error: std::fmt::Debug + std::fmt::Display + Send + Sync + 'static,
+    Msg: Serialize + Debug,
+    Error: Debug + Display + Send + Sync + 'static,
 {
     funds: &'a [Coin],
     contract_addr: &'a Addr,
     msg: Msg,
     app: &'app App,
```

Comment:
This `app` type is invalid - it is the default app type, so it never supports custom queries/messages. It has to be generic.